spec/schemas.spec.js

```diff
@@ -1573,4 +1569,40 @@ describe('schemas', () => {
       });
     });
   });
+
+  fit("regression test for #1991", done => {
```

Comment:
Should be `it`. Travis seems to have run every test anyway, though, so I will just merge and then fix this.